### DISCUSSION TITLE
Welcome to chaiNNer Discussions!

### DISCUSSION THREAD

**Discussion by joeyballentine** [chaiNNer-org/chaiNNer#2024] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mcwv** [comment]
I'm wondering what the 'recursive' tick box does

**Comment by throwaway1908** [comment]
Hi, when it comes to video upscaling, is there an option to change the fps of the video? I can't seem to find it if there is.
